### PR TITLE
Fixed language translate

### PR DESCRIPTION
**Pull request by C-Dao**

### CODE REVIEW

File: src/app/pages/SettingPage.tsx [hunk 1]
```diff
@@ -138,7 +138,7 @@ function SettingPage() {
           {userConfig.chatgptMode === ChatGPTMode.API ? (
             <div className="flex flex-col gap-2">
               <div className="flex flex-col gap-1">
-                <p className="font-medium text-sm">API Key</p>
+                <p className="font-medium text-sm">API {t('Key')}</p>
```
Comment (wong2) [inline src/app/pages/SettingPage.tsx:141]: I don't think this need translation, so does Host, Model

Comment (C-Dao) [reply src/app/pages/SettingPage.tsx:141]: ok，I will revert it

**Comment by wong2** [comment]
why is th lang removed?

**Comment by wong2** [comment]
Please, don't do extra works, just update the existing translations.

**Comment by C-Dao** [comment]
> Please, don't do extra works, just update the existing translations.

Why? there are some words never translated, I think them need to be translated.

**Comment by wong2** [comment]
Some of them are not necessary to translate, for example "prompt".

**Comment by C-Dao** [comment]
> Some of them are not necessary to translate, for example "prompt". 

Ok, I revert the last commit

**Comment by wong2** [comment]
Thanks

**Pull request merged**